ruby/README.md

```diff
@@ -0,0 +1,79 @@
+# Ruby
+
+- Avoid conditional modifiers (lines that end with conditionals).
+- Avoid multiple assignments per line (`one, two = 1, 2`).
+- Avoid organizational comments (`# Validations`).
+- Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
+  instead to emphasize code branches.
+- Avoid bang (!) method names. Prefer descriptive names. [#122]
+- Name variables created by a factory after the factory (`user_factory` creates
+  `user`).
+- Prefer nested class and module definitions over the shorthand version
+- Prefer `detect` over `find`.
+- Prefer `select` over `find_all`.
+- Prefer `map` over `collect`.
+- Prefer `reduce` over `inject`.
+- Prefer `&:method_name` to `{ |item| item.method_name }` for simple method
+  calls.
+- Use `_` for unused block parameters.
+- Prefix unused variables or parameters with underscore (`_`).
+- Suffix variables holding a factory with `_factory` (`user_factory`).
+- Use a leading underscore when defining instance variables for memoization.
+- Use `%()` for single-line strings containing double-quotes that require
+  interpolation.
+- Use `?` suffix for predicate methods.
+- Use `def self.method`, not `class << self`.
+- Use `def` with parentheses when there are arguments. 
+- Use heredocs for multi-line strings.
+- Order class methods above instance methods.
+- Prefer method invocation over instance variables.
+- Avoid optional parameters. Does the method do too much?
+- Avoid monkey-patching.
+- Generate necessary [Bundler binstubs] for the project, such as `rake` and
+  `rspec`, and add them to version control.
+- Prefer classes to modules when designing functionality that is shared by
+  multiple models.
+- Prefer `private` when indicating scope. Use `protected` only with comparison
+  methods like `def ==(other)`, `def <(other)`, and `def >(other)`.
+
+[bundler binstubs]: https://github.com/sstephenson/rbenv/wiki/Understanding-binstubs
+
+## Bundler
+
+- Specify the [Ruby version] to be used on the project in the `Gemfile`.
+- Use a [pessimistic version] in the `Gemfile` for gems that follow semantic
+  versioning, such as `rspec`, `factory_bot`, and `capybara`.
+- Use a [versionless] `Gemfile` declarations for gems that are safe to update
+  often, such as pg, thin, and debugger.
+- Use an [exact version] in the `Gemfile` for fragile gems, such as Rails.
+
+[ruby version]: http://bundler.io/v1.3/gemfile_ruby.html
+[exact version]: http://robots.thoughtbot.com/post/35717411108/a-healthy-bundle
+[pessimistic version]: http://robots.thoughtbot.com/post/35717411108/a-healthy-bundle
+[versionless]: http://robots.thoughtbot.com/post/35717411108/a-healthy-bundle
+
+## Ruby Gems
+
+- Declare dependencies in the `<PROJECT_NAME>.gemspec` file.
+- Reference the `gemspec` in the `Gemfile`.
+- Use [Appraisal] to test the gem against multiple versions of gem dependencies
```

Comment:
Ah this section is for actual gems we have created, NVM.

Reply:
Yeah, I debated keeping this Ruby Gems section in or not. I ended up keeping it here because we DO have at least one gem with some others potentially in the works.

Reply:
I like the section! I see us only having more in the future as well.